.gitignore

```diff
@@ -12,3 +12,5 @@
 /.dockerignore
 .ycm_extra_conf.py
 apex
+config/
+runtime/parcelset/
```

Comment:
Why have these directories been added to the .gitignore file?

Reply:
This file has been accidentally committed.I will soon remove it